### PR TITLE
core-1636 Go to output folder in dashboard and landing page.

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

File: src/components/dashboard/dashboardItem/AnalysisItem.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ class AnalysisItem extends ItemBase {
                 >
                     <Link
                         href={`/${NavConstants.DATA}/ds/[...pathItems]`}
-                        as={getFolderPage(item.content["result_folder_path"])}
+                        as={getFolderPage(item.content["resultfolderid"])}
```
Comment (psarando) [inline src/components/dashboard/dashboardItem/AnalysisItem.js:75]:
```suggestion
                        as={getFolderPage(item.content.resultfolderid)}
```

**Review by psarando — APPROVED** [review]
LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**